gke/README.md

```diff
@@ -143,12 +143,18 @@ Use [Apigee](https://cloud.google.com/apigee) for API key management, throttling
 
 - Open GCP console
 - Navigate to Apigee console
-- Click "Enable" button (wait ~1 hour to complete)
+- Click "Customize your setup" button
+- Follow steps, ensuring that apigee is configured to run in the same region as your GKE cluster (setup takes ~1 hour to complete)
+- Use the apigee console to create a new "Instance" in the same region as your GKE cluster. Ensure logging is enabled for the instance.
+- Use the apigee console to create a new "Environment Group" (navigate to "Environment" section the choose "Environment Group"). Add applicable hostnames to the environment.
```

Comment:
navigate to "Environment" section, then choose ...

Reply:
fixed